tests/integration/test_actions.py

```diff
@@ -62,27 +62,55 @@ async def test_list_action(self, database, setup_providers, caplog):
             assert record.message == line
 
     @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "config,files",
+        [
+            (None, ["mrack-inventory.yaml", "pytest-multihost.yaml"]),
+            (
+                ["ansible-inventory", "pytest-mh"],
+                ["mrack-inventory.yaml", "pytest-mh.yaml"],
+            ),
+            (
```

Comment:
This test looks as duplicate of previous. Did you meant to use `pytest-multihost` instead of `pytest-mh` ?

Reply:
Yes.